### ISSUE TITLE
call custom controller methods

### ISSUE THREAD

**Issue by dschick** [sardana-org/sardana#975]
I guess that this is a rather stupid question with a simple answer:
I am using the `SmaractMCSCtrl `from the Sardana Controller Repsoitory.
It implements a custom method called `SendToCtrl `to send certain commands to the controller, which are not implemented in the standard Sardana motor controller, e.g. a Homing command.

I am simply not able to call that custom method from within Spock.

Any help?

Thanks

Daniel

**Comment by reszelaz** [comment]
Yes, the macro seems like the most convenient way of using this feature from spock.
From the macro code you can see that the interface is via the Pool's `SendToController` command.
